src/main/kotlin/step3/RoundResult.kt

```diff
@@ -0,0 +1,19 @@
+package step3
+
+data class RoundResult(
+    val carPositions: List<CarPosition>
+) {
+    data class CarPosition(
```

Comment:
생성자에 대한 접근 제어자도 신경 써주시면 좋을 거 같아요~

```suggestion
    data class CarPosition private constructor(
```

Reply:
와!! 확실히 `private constructor` 가 맞겠네요~ 아무래도 `CarPosition` 이 자동차에 대한 위치 정보니까 
정적 팩터리 메서드 `from(car: Car)` 통해서만 생성 될 것 같아요!
코멘트 감사합니다.👍👍